index.html

```diff
@@ -18,8 +18,16 @@
     </nav>
```

Comment:
- [ ] I suggest not using a nav tag around the logo, as you already have around the hamburger and the desktop nav list :recycle:

Reply:
removed the nav tag on the logo.

Reply:
- As required by the previous reviewer, kindly remove the logo from the nav and place it before the nav tag to implement a good use of the semantic tag.